go.mod

```diff
@@ -32,10 +32,11 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/hashicorp/go-version v1.3.0
+	github.com/joho/godotenv v1.4.0
 	github.com/lib/pq v1.9.0
 	github.com/minio/minio-go/v7 v7.0.10
 	github.com/percona-platform/dbaas-api v0.0.0-20211201151251-014259873599
-	github.com/percona-platform/saas v0.0.0-20211101203847-f65c32bc8770
+	github.com/percona-platform/saas v0.0.0-20220114102220-b44011a3c642
```

Comment:
updated with `go get -v github.com/percona-platform/saas@PMM-9265-telemetry-v2`
pointing now to `PMM-9265-telemetry-v2` branch.

TODO: after https://github.com/percona-platform/saas/pull/55